src/highlight.js

```diff
@@ -515,10 +515,34 @@ https://highlightjs.org/
    * @property {string} language - the language name
    * @property {number} relevance - the relevance score
    * @property {string} value - the highlighted HTML code
+   * @property {string} originalCade - the original raw code
```

Comment:
It seems that `originalCode` may be misleading, because a before plugin can change it and it will no longer be "original". Why not just `code` (or `codeToHighlight`)?

Reply:
> because a before plugin can change it and it will no longer be "original".

Look again.  A plugin can't change it.

Reply:
LOL, n/m. hmmmm

Reply:
// the plugin can change the desired language or the code to be highlighted
     // just be changing the object it was passed
     fire("before:highlight", context);

Am I missing something?

Reply:
No, that's why I said "n/m".  I looked again. You're right.  I think we can come up with better naming as you suggest.

Reply:
Resolved naming.